src/librustc_plugin/registry.rs

```diff
@@ -103,32 +89,6 @@ impl<'a> Registry<'a> {
         self.register_syntax_extension(Symbol::intern(name), ext);
     }
 
-    /// Register a compiler lint pass.
```

Comment:
Why did you remove these functions? This seems to break plugins registering lints?

Reply:
We publicly expose the lint store now, and it should be a simple transform to use `registry.lint_store.register` vs these functions. Plugins needed to change anyway as we've changed the signature here and expect separate registration of lints and lint passes.